### PR TITLE
add subscriptionData to orders open api schema

### PR DESCRIPTION
**Pull request by DannyC-BC**
#### Types of changes
- [X] New content (endpoints, descriptions or fields from scratch)
- [ ] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [ ] No, but I am going to.

### CODE REVIEW

File: VTEX - Orders API.json [hunk 1]
```diff
@@ -75,6 +75,77 @@
                                     "title": "",
                                     "type": "object",
                                     "properties": {
+                                        "subscriptionData": {
+                                            "type": "object",
```
Comment (karenkrieger) [inline VTEX - Orders API.json:79]: Add `description`

File: VTEX - Orders API.json [hunk 2]
```diff
@@ -75,6 +75,77 @@
...
+                                                "items": {
+                                                  "type": "object",
+                                                  "properties": {
+                                                    "ExecutionCount": {
+                                                      "type": "integer",
+                                                      "format": "int32",
+                                                      "nullable": true
+                                                    },
+                                                    "PriceAtSubscriptionDate": {
+                                                      "type": "string",
```
Comment (karenkrieger) [inline VTEX - Orders API.json:97]: Add `description`

File: VTEX - Orders API.json [hunk 3]
```diff
@@ -75,6 +75,77 @@
...
+                                                      "nullable": true
+                                                    },
+                                                    "ItemIndex": {
+                                                      "type": "integer",
+                                                      "format": "int32",
```
Comment (karenkrieger) [inline VTEX - Orders API.json:102]: Add `description`

File: VTEX - Orders API.json [hunk 4]
```diff
@@ -75,6 +75,77 @@
...
+                                                      "nullable": true
+                                                    },
+                                                    "Plan": {
+                                                      "type": "object",
```
Comment (karenkrieger) [inline VTEX - Orders API.json:106]: Add `description`

File: VTEX - Orders API.json [hunk 5]
```diff
@@ -75,6 +75,77 @@
...
+                                                      "nullable": true,
+                                                      "properties": {
+                                                        "type": {
+                                                          "type": "string",
+                                                          "nullable":true
```
Comment (karenkrieger) [inline VTEX - Orders API.json:111]: Add `description`

File: VTEX - Orders API.json [hunk 6]
```diff
@@ -75,6 +75,77 @@
...
+                                                        },
+                                                        "frequency": {
+                                                          "type": "object",
+                                                          "nullable":true,
```
Comment (karenkrieger) [inline VTEX - Orders API.json:115]: Add `description`

File: VTEX - Orders API.json [hunk 7]
```diff
@@ -75,6 +75,77 @@
...
+                                                          "properties": {
+                                                            "periodicity": {
+                                                              "type": "string",
+                                                              "nullable": true
```
Comment (karenkrieger) [inline VTEX - Orders API.json:119]: Add `description`

File: VTEX - Orders API.json [hunk 8]
```diff
@@ -75,6 +75,77 @@
...
+                                                            },
+                                                            "interval": {
+                                                              "type": "integer",
+                                                              "format": "int32",
+                                                              "nullable": true
```
Comment (karenkrieger) [inline VTEX - Orders API.json:124]: Add `description`

File: VTEX - Orders API.json [hunk 9]
```diff
@@ -75,6 +75,77 @@
...
+                                                            }
+                                                          }
+                                                        },
+                                                        "validity": {
+                                                          "type": "object",
```
Comment (karenkrieger) [inline VTEX - Orders API.json:129]: Add `description`

File: VTEX - Orders API.json [hunk 10]
```diff
@@ -75,6 +75,77 @@
...
+                                                          "nullable": true,
+                                                          "properties": {
+                                                            "begin": {
+                                                              "type": "string",
+                                                              "nullable": true
```
Comment (karenkrieger) [inline VTEX - Orders API.json:134]: Add `description`

File: VTEX - Orders API.json [hunk 11]
```diff
@@ -75,6 +75,77 @@
...
+                                                            },
+                                                            "end": {
+                                                              "type": "string",
+                                                              "nullable": true
```
Comment (karenkrieger) [inline VTEX - Orders API.json:138]: Add `description`

**Review by karenkrieger — CHANGES_REQUESTED** [review]
Thanks for contributing to our API documentation! ✨ I left some comments in the review to align the code with our API documentation standards

**Comment by julia-rabello** [comment]
@Isabella-Veloso this is an external contribution from last year, adding `subscriptionData` information to the [Get order](https://developers.vtex.com/docs/api-reference/orders-api#get-/api/oms/pvt/orders/-orderId-) endpoint. Can you review these proposed changes and contribute with the missing descriptions for this to be published?

**Comment by Isabella-Veloso** [comment]
This PR deprecated, since the documentation has already been updated with this content.

**Pull request closed without merging**